### ISSUE TITLE
groups `r.meta is undefined` error message when opening groups

### ISSUE THREAD

**Issue by matthiasschaub** [tloncorp/landscape-apps#2615]
This happened right after leaving a group and hold on for a couple of minutes. After a short time, every thing continued to work as expected.
```
Cl@https://myship.com/apps/groups/assets/index.b18819a0.js:48:175993
Kl@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884
Ka@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:62361
mf@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:9604
pf@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:949
yg@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:877
rn@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:730
sl@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:10884
Ud/<@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38716
e.unstable_runWithPriority@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:8:3811
_r@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38494
Ud@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38664
ir@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38597
br@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:8455
Zl@https://myship.com/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:58182
hN/</<@https://myship.com/apps/groups/assets/index.b18819a0.js:48:318484
```

**Comment by jamesacklin** [comment]
Fixed by #2635

**Comment by jamesacklin** [comment]
Tracking internally as LAND-587
